src/components/structures/MatrixChat.js

```diff
@@ -139,6 +139,10 @@ module.exports = React.createClass({
         return this.props.config.default_is_url || "https://vector.im";
     },
 
+    getEnableLabs() {
+        return this.props.config.enableLabs === undefined ? true : this.props.config.enableLabs;
```

Comment:
Well, just having it shown doesn't turn any options on, so I figured leave it on by default, but really happy to do either to be honest.